### ISSUE TITLE
Empty folder

### ISSUE THREAD

**Issue by vkjansenac** [timsutton/brigadier#46]
Hi, 
I try to download the drivers for my iMac 27'' late 2013 for W10: The program shows download actity and then ends without error message but leaves the download folder empty.
What's wrong?

**Comment by Welfordian** [comment]
Looks like you need to install 7Zip.

**Comment by vectorcr** [comment]
Yeah I just headed over to https://www.7-zip.org/download.html and downloaded the 64bit version and upon rerunning the tool it worked!
